### DISCUSSION TITLE
using /grab decreased my xp instead?

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#410] [Q&A] [answered]
I recently tried /grab, but after I start the ship, I got my xp taken instead of given. it works like like minus points when we failed on a planet.

**Comment by Dexteriority** [comment]
Also /xp says command not found

**Reply by winstxnhdw** [reply]
![image](https://github.com/winstxnhdw/lc-hax/assets/56998716/e7043f52-1644-43a0-b01f-23d49a5c5959)

Check if you have `ExperienceCommand.cs` and if you see this header at the top.

**Reply by Dexteriority** [reply]
I have it, but it says here
![image](https://github.com/winstxnhdw/lc-hax/assets/157716300/8e670204-6d63-45b9-b383-b051d6303477)

I used a mod to add another save files, maybe it affects this command?

**Reply by winstxnhdw** [reply]
Wow. That mod must be doing some insane patching to completely disable one of my commands.

**Reply by Dexteriority** [reply]
no its not, sorry.
I just re-clone the repository, used a Vanilla. and it still says the same.

**Reply by winstxnhdw** [reply]
I’d bet BepinEx broke your game files.

**Reply by Dexteriority** [reply]
Oh I see..

**Reply by winstxnhdw** [reply]
Nvm, I am an idiot. I think it's actually broken.

**Reply by winstxnhdw** [reply, marked as answer]
Okay, pushed in latest. Fixed.

**Reply by Dexteriority** [reply]
Oh! Thank you!
